.ruby-version

```diff
@@ -1 +0,0 @@
-2.1.3
```

Comment:
Yeah, I was wrong about that. However, if you only rely on the Gemfile for specifying the Ruby version, you will see this warning every time you `cd` into the project (unless you disable the warning per the instructions):
```
RVM used your Gemfile for selecting Ruby, it is all fine - Heroku does that too,
you can ignore these warnings with 'rvm rvmrc warning ignore /Users/moncefbelyamani/projects/18f/save-ferris/Gemfile'.
To ignore the warning for all files run 'rvm rvmrc warning ignore allGemfiles'.
```